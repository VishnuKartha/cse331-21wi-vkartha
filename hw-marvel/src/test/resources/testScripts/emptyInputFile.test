#Tries to find a path of a graph loaded from an empty tsv file

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/empty.tsv


LoadGraph empty empty.tsv

FindPath empty VISHNU FRIEND_OF_VISHNU
FindPath empty VISHNU VISHNU
