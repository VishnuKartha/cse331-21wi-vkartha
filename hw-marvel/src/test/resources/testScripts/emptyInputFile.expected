#Tries to find a path of a graph loaded from an empty tsv file

# To run this test, the marvel dataset must exist at
# hw-marvel/src/main/resources/data/empty.tsv


loaded graph empty

unknown character VISHNU
unknown character FRIEND OF VISHNU
unknown character VISHNU
unknown character VISHNU